packages/protocol/migrations/08_stabletoken.ts

```diff
@@ -54,12 +54,22 @@ module.exports = deploymentForCoreContract<StableTokenInstance>(
       await stableToken.mint(address, initialBalance)
     }
 
-    console.log('Setting GoldToken/USD exchange rate')
     const sortedOracles: SortedOraclesInstance = await getDeployedProxiedContract<
       SortedOraclesInstance
     >('SortedOracles', artifacts)
 
-    await sortedOracles.addOracle(stableToken.address, minerAddress)
+    for (const oracle of config.stableToken.priceOracleAccounts) {
+      console.log(`Adding ${oracle} as an Oracle for StableToken`)
```

Comment:
Small nit: it's generally preferred to choose a log level when logging. i.e. `console.debug` or `console.info` instead of `console.log`
I guess we don't have lint checks here or it would have caught that

Reply:
Every migration file that has logging uses `console.log`. I'm happy to change all instances of those, but it makes me wonder if there's a reason for it.